typings/resin-corvus/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'sudo-prompt';
```

Comment:
Huh, `typings/resin-corvus/index.d.ts` and `typings/sudo-prompt/index.d.ts` **both** say
```typescript
declare module 'sudo-prompt';
```
??? :confused: